KnowWhereGraph/kwg-faceted-search
# Add the map to the 'Persons' tab

Related to #279

This PR adds map functionality for the 'Persons' tab.

Notes:
1. We're now loading map assets locally
2. I added contact information to the cards, when available (changes to the query and table component)

To test:
1. Checkout
2. Run locally
3. Visit 'Persons' tab
4. Click around on the map, filter results, make sure results clear when clicking other tabs



![image](https://user-images.githubusercontent.com/9289652/230792175-29b5caec-438f-4c9d-ad3d-73c6a2361a4c.png)
![persons1](https://user-images.githubusercontent.com/9289652/230792350-4589b315-5cef-4daa-a9fe-eabcf8265822.png)

![persons2](https://user-images.githubusercontent.com/9289652/230792352-ddcbd66c-5f2a-46b4-99e2-000826a78f36.png)
